### PR TITLE
remove mdc_web_sass package in favor of mdc_web package

### PR DESCRIPTION
**Pull request by johnpryan**
official mdc_web package now includes the sass files from material-components-web 
https://github.com/jifalops/mdc_web/pull/4

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -667,4 +660,4 @@ packages:
     source: hosted
     version: "2.1.16"
 sdks:
-  dart: ">=2.4.0 <3.0.0"
+  dart: ">=2.3.0 <3.0.0"
```
Comment (RedBrogdon) [inline pubspec.lock]: Why the downgrade here?

Comment (johnpryan) [reply pubspec.lock]: Hmm.. I've been using the 2.4.0 SDK... It seems like pub might be using the largest range possible (consistent with the SDK constraints for all the packages)

**Comment by johnpryan** [comment]
@RedBrogdon PTAL

**Comment by johnpryan** [comment]
@RedBrogdon the latest commits are just updating with master, merging.

**Pull request merged**